src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public static Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
make this instance variable, no need to create new instance per each method invocation